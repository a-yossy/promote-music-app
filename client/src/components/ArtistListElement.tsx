import { FC } from "react";
import { Artist } from "lib/artist";

type ArtistListElementProps = {
  artist: Artist
};

const ArtistListElement: FC<ArtistListElementProps> = ({ artist }) => {
  return (
    <div>
      {artist.name}
    </div>
  )
};

export default ArtistListElement;
