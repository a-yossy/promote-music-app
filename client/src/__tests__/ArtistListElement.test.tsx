import rerender from 'react-test-renderer';
import ArtistListElement from 'components/ArtistListElement';

it('render correcyly', () => {
  const tree = rerender
    .create(<ArtistListElement artist={{ id: 1, name: 'artist' }} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
