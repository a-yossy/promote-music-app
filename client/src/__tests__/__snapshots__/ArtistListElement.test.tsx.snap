// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correcyly 1`] = `
<div>
  artist
</div>
`;
